packages/simulator/index.js

```diff
@@ -56,12 +58,27 @@ const engine = {
       share()
     )
 
+    // TODO: Rename to vehicleUpdates
     experiment.carUpdates = merge(
       experiment.cars,
       experiment.buses,
       experiment.taxis
     ).pipe(
       mergeMap((car) => fromEvent(car, 'moved')),
+      map((car) => {
+        experiment.kommuner.forEach((kommun) => {
+          if (isInsideCoordinates(car.position, kommun.geometry.coordinates)) {
+            console.log(
+              `Vehicle (${car.vehicleType}) ${car.id} is in ${kommun.name}`
+            )
+            // NOTE: add car co2 to kommun
+            // TODO: fix this, it adds too much
+            kommun.co2 += car.co2
```

Comment:
This is bad, it does not add the increment it adds the vehicle's total each time!